docs/docs/PythonUserGuide/install-from-pip.md

```diff
@@ -1,9 +1,10 @@
 ## **NOTES**
 
 - Pip install supports __Mac__ and __Linux__ platforms.
-- This package is built with JDK7 by default, so you need to install Java __>= JDK7__ before runing BigDL.
+- This package is built with __JDK7__ by default, so you need to install Java __>= JDK7__ before runing BigDL.
```

Comment:
install Java __>= JDK8__ before runing BigDL which is required by pyspark

Reply:
Modified.